packages/p2p/src/components/app.jsx

```diff
@@ -19,6 +20,25 @@ import MyProfile from './my-profile/my-profile.jsx';
 import './app.scss';
 
 const allowed_currency = 'USD';
+
+const TemporaryBarredMessage = () => {
+    const { general_store } = useStores();
+
+    const formatted_date = getFormattedDateString(new Date(convertToMillis(general_store?.user_blocked_until)));
+
+    return (
+        <Toast className='p2p-barred-user' is_open={general_store.is_barred}>
```

Comment:
Should we use this for both desktop and mobile?

Reply:
Yep.

Reply:
What does it do for desktop?

Reply:
The barred user message is visible for both mobile and desktop.